waku/v2/protocol/waku_rln_relay/waku_rln_relay_utils.nim

```diff
@@ -1183,3 +1201,15 @@ proc mountRlnRelay*(node: WakuNode, conf: WakuNodeConf|Chat2Conf, spamHandler: O
       if res.isErr:
         return err("dynamic rln-relay could not be mounted: " & res.error())
       return ok(true)
+
+
+proc mountRlnRelay*(node: WakuNode, conf: WakuNodeConf|Chat2Conf, spamHandler: Option[SpamHandler] = none(SpamHandler), registrationHandler: Option[RegistrationHandler] = none(RegistrationHandler)): RlnRelayResult[bool] {.raises: [Defect, ValueError, IOError, CatchableError, Exception].} =
+  waku_rln_relay_mounting_time.time:
```

Comment:
For interest sake, if we are splitting the `mount` proc just to measure the time to mount, what are the individual actions during mounting that we believe will take a long time? Registration? Would it make sense to measure these items individually rather than the more vague "mount time"?

Reply:
You're right! registration is will be where most time is spent, While this may not necessarily be an important metric, we're measuring broadly the time to mount the relay, to see how it would affect the overall node startup time (IIUC). Unsure if we're measuring total node setup time.

cc: @staheri14

Reply:
Thanks for the comment @jm-clius. As the mounting procedure is done only once, we are mostly interested its overall running time. Yet, I agree we can have more fine-grained metrics for other specific parts. However, the registration is somewhat out of the control of the rln-relay as it is impacted by the underlying Blockchain traffic and the network throughput, thus even if we know how long it takes, there is not much we can do about it. Nevertheless, it would be good to understand what portion of the setup time is due to registration or other internal mechanics of the rln-relay.
@rymnc please do not delete the overall mounting time, but please please add the following new metrics to the PR:
- The registration time 
- The time required to create an instance of an RLN (it is to measure the Merkle tree performance)
- The time required for the insertion of a new commitment to the tree, with the index of the commitment (it is to measure the Merkle tree performance)
- The time for importing credentials from the file

Reply:
Addressed in [676c689](https://github.com/status-im/nwaku/pull/1185/commits/676c689b4afaaac698a159eb4b371e750c704e1c)